thunder/dynamo/report.py

```diff
@@ -515,13 +515,16 @@ def write_repro(
             print(code_str, file=f)
         format_python_file(folder / file_name)
 
-    def run_benchmark(self, compile_fn: CompileSpecificationInterface, time_fn: TimerInterface):
+    def run_benchmark(
+        self, compile_fn: CompileSpecificationInterface, time_fn: TimerInterface, reset_torch_dynamo=True
```

Comment:
Nit: `*, reset_torch_dynamo: bool = True` -- might as well make it kwarg-only for readability